TrainingApp/Utils/Extensions.swift

```diff
@@ -316,3 +450,13 @@ class PrivateBundle: Bundle {
                 (super.localizedString(forKey: key, value: value, table: tableName))
     }
 }
+
+extension UITableView {
+    func deselectAllRows(animated: Bool) {
+        guard let selectedRows = indexPathsForSelectedRows else { return }
+        print(selectedRows)
```

Comment:
Remove print() and add space after guard let else { return }

Reply:
Done 👍